cmd/main.go

```diff
@@ -44,27 +45,31 @@ import (
 	sdkMetrics "github.com/redhat-cne/sdk-go/pkg/localmetrics"
 	v1amqp "github.com/redhat-cne/sdk-go/v1/amqp"
 	v1event "github.com/redhat-cne/sdk-go/v1/event"
+	v1http "github.com/redhat-cne/sdk-go/v1/http"
 	v1pubs "github.com/redhat-cne/sdk-go/v1/pubsub"
 )
 
 var (
 	// defaults
-	storePath         string
-	amqpHost          string
-	apiPort           int
-	channelBufferSize = 100
-	scConfig          *common.SCConfiguration
-	metricsAddr       string
-	apiPath           = "/api/cloudNotifications/v1/"
+	storePath          string
+	transportHost      string
+	apiPort            int
+	channelBufferSize  = 100
+	scConfig           *common.SCConfiguration
+	metricsAddr        string
+	apiPath            = "/api/cloudNotifications/v1/"
+	httpEventPublisher string
+	pluginHandler      plugins.Handler
 )
 
 func main() {
 	// init
 	common.InitLogger()
 	flag.StringVar(&metricsAddr, "metrics-addr", ":9091", "The address the metric endpoint binds to.")
 	flag.StringVar(&storePath, "store-path", ".", "The path to store publisher and subscription info.")
-	flag.StringVar(&amqpHost, "transport-host", "amqp:localhost:5672", "The transport bus hostname or service name.")
+	flag.StringVar(&transportHost, "transport-host", "amqp:localhost:5672", "The transport bus hostname or service name.")
```

Comment:
How about set default value to `localhost:9043` for local http testing?
`flag.StringVar(&transportHost, "transport-host", "localhost:9043", "The transport bus hostname or service name.")`

Reply:
For http it should never set to localhost, it should be k8s service url